featuretools/tests/primitive_tests/test_features_deserializer.py

```diff
@@ -3,7 +3,14 @@
 import pandas as pd
 import pytest
 
-import featuretools as ft
+from featuretools import (
+    AggregationFeature,
+    Feature,
+    IdentityFeature,
+    TransformFeature,
+    __version__,
+    primitives,
```

Comment:
I suggest importing the primitive we need directly instead of importing the entire primitives module.